THIRD-PARTY-NOTICES

```diff
@@ -390,4 +390,17 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
-https://github.com/System-IO-Abstractions/System.IO.Abstractions/blob/master/LICENSE
\ No newline at end of file
+https://github.com/System-IO-Abstractions/System.IO.Abstractions/blob/master/LICENSE
+
+
+License notice for BouncyCastle
+-----------------------------------------------------
+Copyright (c) 2000 - 2017 The Legion of the Bouncy Castle Inc. (http://www.bouncycastle.org)
```

Comment:
I think this should be (c) 2000 - 2020 (see http://www.bouncycastle.org/licence.html)

Reply:
Huh, it's different than the one for C# (http://www.bouncycastle.org/csharp/licence.html)
I'll take the one you pointed to.